src/commons/utils/Hooks.ts

```diff
@@ -118,8 +118,11 @@ export const useDimensions = (ref: RefObject<HTMLElement>): [width: number, heig
  * or desktop as defined by the constants file.
  */
 export const useResponsive = () => {
-  const isMobileBreakpoint = useMediaQuery({ maxWidth: Constants.mobileBreakpoint });
-  return { isMobileBreakpoint };
+  const isMobileBreakpoint = useMediaQuery(`(max-width: ${Constants.mobileBreakpoint}px)`);
+  return {
+    isMobileBreakpoint: !!(isMobileBreakpoint ?? false),
+    isDesktopBreakpoint: !(isMobileBreakpoint ?? true)
```

Comment:
> is !! necessary?
Missed out on that one, will remove.

As for the desktop breakpoint, there seems to be some issues taking the negation, since during the first render the output value of `isMobileBreakpoint` should be `undefined` (a falsy value), and taking the negation of that results in a truthy value which breaks some of the snapshots for `AssessmentWorkspace` (which assumes that the initial render does not render the component in `ReplInput.tsx`).